CMakeLists.txt

```diff
@@ -106,21 +105,18 @@ SET(Boost_ADDITIONAL_VERSIONS "1.67" "1.67.0")
 
 SET(BOOST_PYTHON_MODULE_NAME "python")
 if ( USE_PYTHON_VERSIONS VERSION_GREATER 3 )
-    if ( APPLE )
-        SET( BOOST_PYTHON_MODULE_NAME ${MACOS_USE_PYTHON_MODULE} )
-    else()
-        SET( BOOST_PYTHON_MODULE_NAME "python3" )
-    endif()
+  SET( BOOST_PYTHON_MODULE_NAME ${BOOST_PYTHON_NAME} )
 endif()
 
 if( WIN32 )
   SET(Boost_USE_STATIC_LIBS ON)
-  find_package( Boost COMPONENTS chrono date_time filesystem iostreams program_options ${BOOST_PYTHON_MODULE_NAME} regex system thread zlib REQUIRED )
+  find_package( Boost COMPONENTS chrono date_time filesystem iostreams program_options ${BOOST_PYTHON_MODULE_NAME} regex system thread REQUIRED )
```

Comment:
Is it safe to drop zlib? It's required by the mission recording code.

Reply:
If it's part of libmalmo, then it should be safe. We have a find_package() for zlib independently at https://github.com/Microsoft/malmo/blob/master/CMakeLists.txt#L145

I tested this with boost 1.66 and boost 1.67 and neither of them had a boost zlib 'component'. Probably boost:iostream required zlib (externally) for some functionality and was linking to it correctly.